lib/game.rb

```diff
@@ -12,13 +12,17 @@ def initialize(ui)
   def play(players, board)
     current_player = players[Marks::X]
     until board.game_over? || !current_player(players, board).ready?
-      ui.draw_board(board)
-      board = current_player(players, board).make_move(board)
+      board = play_one_round(board, players)
     end
     end_game(board) if board.game_over?
     board
   end
 
+  def play_one_round(board, players)
+      ui.draw_board(board)
+      current_player(players, board).make_move(board)
+  end
+
   def current_player(players, board)
     players[current_player_mark(board)]
```

Comment:
`current_players_marks` has no dependencies other than `Board` and could be moved there.
Then this would become `players[board.currentPlayer]` or `players[board.nextPlayerMark]`.